eiffel-vocabulary/EiffelIssueDefinedEvent.md

```diff
@@ -1,10 +1,10 @@
 <!---
-   This file was generated from ../definitions/EiffelIssueDefinedEvent/3.1.0.yml.
+   This file was generated from ../definitions/EiffelIssueDefinedEvent/3.2.0.yml.
    See that file for a copyright notice.
 --->
 
 # EiffelIssueDefinedEvent (ID)
-The EiffelIssueDefinedEvent declares that an issue has been created in some external issue management software. ID is semantically similar to [EiffelFlowContextDefinedEvent](../eiffel-vocabulary/EiffelFlowContextDefinedEvent.md) and [EiffelEnvironmentDefinedEvent](../eiffel-vocabulary/EiffelEnvironmentDefinedEvent.md).
+The EiffelIssueDefinedEvent declares that an issue has been created in someexternal issue management software. ID is semantically similar to [EiffelFlowContextDefinedEvent](../eiffel-vocabulary/EiffelFlowContextDefinedEvent.md) and [EiffelEnvironmentDefinedEvent](../eiffel-vocabulary/EiffelEnvironmentDefinedEvent.md).
```

Comment:
This file also hasn't been updated to include the last commits from the PR branch that introduced the YAML files.